src/main/java/duke/utility/parser/CommandMap.java

```diff
@@ -0,0 +1,12 @@
+package duke.utility.parser;
+
```

Comment:
Consider removing the blank like at line 2 to ensure consistency across code files..